tc-tests-utils.sh

```diff
@@ -367,6 +367,15 @@ install_pyenv_virtualenv()
   eval "$(pyenv virtualenv-init -)"
 }
 
+maybe_install_xldd()
```

Comment:
The RPi3 specific toolchain includes a hacked ldd somehow, but that is not officially supported. Instead, switch to the usual, crosstool-ng provided, xldd solution. We rely on `print-toolchain` to avoid having to duplicate the same information over and over.